### DISCUSSION TITLE
buscando equipo para la hackaton

### DISCUSSION THREAD

**Discussion by wilver112** [Launch-X-Latam/MisionFrontEnd#564] [Team Posts]
No tengo mucha experencia pero si muchas ganas de aprender y desarrollar los conocimientos adquiridos, me estoy poniendo a punto con las clases, y dispuesto a seguir aprendiendo. Busco un equipo para este gran reto de la hackaton, si alguien me pudiera incluir en su team, mi correo es lx14471@innovaccion.mx 

Saludos
Wilver Quintero

**Comment by CarlNauj** [comment]
Buenos días Estimado:

                Mi nombre es Juan Carreño, soy de Chile. Al igual que tú estoy poniéndome al día (de hecho todavía no termino la práctica del Frontend). Pero tengo toda la disposición a participar, si quieres.

                Atento a tu respuesta.

              Saluda atentamente a usted.

JUAN CARLOS CARREÑO POBLETE.

De: wilver112 ***@***.***>
Enviado el: martes, 15 de noviembre de 2022 02:19
CC: Subscribed ***@***.***>
Asunto: [Launch-X-Latam/mision-backend] buscando equipo para la hackaton (#101)


No tengo mucha experencia pero si muchas ganas de aprender y desarrollar los conocimientos adquiridos, me estoy poniendo a punto con las clases, y dispuesto a seguir aprendiendo. Busco un equipo para este gran reto de la hackaton, si alguien me pudiera incluir en su team, mi correo es ***@***.******@***.***>

Saludos
Wilver Quintero

-
